api/serializers/assets.py

```diff
@@ -130,6 +140,39 @@ def to_internal_value(self, data):
             internals["specs"] = specs
         return internals
 
+    # # allow updating of active_inactive and paid for specific asset_types
+    def update(self, instance, validated_data):
+
+        asset_type = instance.model_number.asset_make.asset_type.name
+        paid = validated_data.get('paid')
+        active_inactive = validated_data.get('active')
```

Comment:
this seems confusing, can you give it a more declarative name, how about `is_active`?

What's the rationale behind the value of this variable?

Reply:
@raheemazeezabiodun That is not part of this PR. I have a commit from develop that has mixed up with my commits...

Reply:
cool cool